tracer/src/instruction/format/format_virtual_i.rs

```diff
@@ -0,0 +1,73 @@
+use crate::emulator::cpu::Cpu;
+use common::constants::REGISTER_COUNT;
+use rand::rngs::StdRng;
+use rand::RngCore;
+use serde::{Deserialize, Serialize};
+use std::fmt::Debug;
+
+use super::{
+    normalize_register_value, InstructionFormat, InstructionRegisterState, NormalizedOperands,
+};
+
+#[derive(Default, Debug, Clone, Copy, PartialEq, Serialize, Deserialize)]
+pub struct FormatVirtualI {
```

Comment:
If this is basically the exact same as `FormatI` let's just use that

Reply:
I've replaced FormatVirtualI with FormatI in MULI instruction, however, there's still a need for a new format - `FormatVirtualRightShiftI` since `random` function has to have specific `imm`, same as in `FormatVirtualRightShift` (renamed to `FormatVirtualRightShiftR`)